cli/daemon/daemon.go

```diff
@@ -88,6 +103,9 @@ func runDaemonCommand(cmd *cobra.Command, args []string) {
 		go func() {
 			// Stdin is closed when the controlling parent process ends
 			_, _ = io.Copy(ioutil.Discard, os.Stdin)
+			if viper.GetBool("telemetry.enabled") {
```

Comment:
do we need this `if`? If telemetry wasn't activated Flush should be a no-op, correct?